spec/controllers/books_controller_spec.rb

```diff
@@ -1,4 +1,11 @@
 require 'rails_helper'
 
 RSpec.describe BooksController, type: :controller do
+  describe "GET #index" do
```

Comment:
INFO: Prefer single-quoted strings when you don't need string interpolation or special symbols.